pkg/terra/config.go

```diff
@@ -1,15 +1,52 @@
 package terra
 
 import (
+	"net/url"
+	"strings"
 	"sync"
 	"time"
 
+	"github.com/pkg/errors"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/smartcontractkit/chainlink-terra/pkg/terra/db"
 )
 
-var DefaultConfigSet = ConfigSet{
+var (
+	mainnetFCDURL *url.URL
+	bombayFCDURL  *url.URL
+)
+
+func init() {
+	var err error
+	mainnetFCDURL, err = url.Parse("https://fcd.terra.dev/v1/txs/gas_prices")
+	if err != nil {
+		panic(errors.Wrap(err, "failed to parse FCD URL"))
+	}
+	bombayFCDURL, err = url.Parse("https://bombay-fcd.terra.dev/v1/txs/gas_prices")
+	if err != nil {
+		panic(errors.Wrapf(err, "failed to parse FCD URL"))
+	}
+}
+
+type ChainID string
+
+// defaults returns Chain specific defaults.
+func (id ChainID) defaults() configSet {
+	c := defaultConfigSet
+	lower := strings.ToLower(string(id))
+	switch {
+	case strings.HasPrefix(lower, "columbus-"):
+		c.FCDURL = *mainnetFCDURL
+	case strings.HasPrefix(lower, "bombay-"):
+		c.FCDURL = *bombayFCDURL
```

Comment:
Perhaps this could default to mainnet for convenience? Could also be risky though, if it surprises someone.

Reply:
hm we already have a fallback gas price, so I'm not sure we also need fallback fcd urls. Could we just check that the URL they provide is a real url at chain add time? Then if its providing invalid data we fallback to our fallback gas price

Reply:
Yeah that makes sense :+1: